docs/source/getting-started.rst

```diff
@@ -14,7 +14,7 @@ Glow requires Apache Spark 2.4.3 (or a later version of Spark 2.4.x that is buil
 
         .. code-block:: sh
 
-          pip install pyspark==2.4.3
+          pip install pyspark==3.0.0
```

Comment:
This doesn't work until we publish a 3.0 artifact, does it?

Reply:
Yes - I'll clarify that this is blocked on the next release.